response_operations_ui/views/reporting_units.py

```diff
@@ -1,8 +1,9 @@
 import logging
 
-from flask import Blueprint, render_template, request
+from flask import Blueprint, render_template, request, url_for
 from flask_login import login_required
 from structlog import wrap_logger
+from werkzeug.utils import redirect
```

Comment:
Why are you using werkzeug utils? Use flask's redirect instead

Reply:
yep - unless your building your own web framework you really don't need to touch werkzeug.